plugins/wpe-headless/includes/settings/callbacks.php

```diff
@@ -229,8 +229,8 @@ function confirm_regenerate_key( event ) {
 	<p class="description">
 		<?php
 		printf(
-			/* translators: %s: link to documentation */
-			wp_kses_post( 'This key is used to enable post previewing with Next.js. Read about post previewing <a href="%s" target="_blank" rel="noopener noreferrer">here</a>.', 'wpe-headless' ),
```

Comment:
IntelliJ was flagging this as "Method call is provided 2 parameters, but the method signature uses 1 parameters" in PhpStorm/IntelliJ, due to `'wpe-headless'` being passed as a second param to `wp_kses_post`.